client/src/annoMatrix/annoMatrix.ts

```diff
@@ -470,10 +471,7 @@ Return cache keys for columns associated with this query.  May return
   /**
    ** Private interfaces below.
    **/
-  _resolveCachedQueries(
-    field: Field,
-    queries: Query[]
-  ): string[] | number[] | [undefined] {
+  _resolveCachedQueries(field: Field, queries: Query[]): string[] | number[] {
```

Comment:
This should return the new Dataframe LabelArray type (see my comment on the definition of WhereCacheColumnLabels)

Reply:
Updated [annoMatrix.ts:_resolveCachedQueries](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L481).